docs/installation.md

```diff
@@ -0,0 +1,152 @@
+use Sylius\Component\Core\Model\Order as BaseOrder;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="sylius_order")
+ */
+#[ORM\Entity]
+#[ORM\Table(name: 'sylius_order')]
+-class Order extends BaseOrder
++class Order extends BaseOrder implements TemporaryAddressesAwareInterface
+{
++    use TemporaryAddressesAwareTrait;
+}
+```
+
+Your `Shipment` entity should implement `MonsieurBiz\SyliusAdvancedShippingPlugin\Entity\AdvancedShipmentMetadataAwareInterface` and use the `MonsieurBiz\SyliusAdvancedShippingPlugin\Entity\AdvancedShipmentMetadataAwareTrait` trait.
+
+```diff
+namespace App\Entity\Shipping;
+
+use Doctrine\ORM\Mapping as ORM;
++use MonsieurBiz\SyliusAdvancedShippingPlugin\Entity\AdvancedShipmentMetadataAwareInterface;
++use MonsieurBiz\SyliusAdvancedShippingPlugin\Entity\AdvancedShipmentMetadataAwareTrait;
+use Sylius\Component\Core\Model\Shipment as BaseShipment;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="sylius_shipment")
+ */
+#[ORM\Entity]
+#[ORM\Table(name: 'sylius_shipment')]
+-class Shipment extends BaseShipment
++class Shipment extends BaseShipment implements AdvancedShipmentMetadataAwareInterface
+{
++    use AdvancedShipmentMetadataAwareTrait;
+}
+```
+
+Your `ShippingMethod` entity should implement `MonsieurBiz\SyliusAdvancedShippingPlugin\Entity\AddressProviderAwareInterface` and `MonsieurBiz\SyliusAdvancedShippingPlugin\Entity\ShippingTypeAwareInterface` interfaces.  
+And use the `MonsieurBiz\SyliusAdvancedShippingPlugin\Entity\AddressProviderAwareTrait` and `MonsieurBiz\SyliusAdvancedShippingPlugin\Entity\ShippingTypeAwareTrait` traits.
+
+```diff
+namespace App\Entity\Shipping;
+
+use Doctrine\ORM\Mapping as ORM;
++use MonsieurBiz\SyliusAdvancedShippingPlugin\Entity\AddressProviderAwareInterface;
++use MonsieurBiz\SyliusAdvancedShippingPlugin\Entity\AddressProviderAwareTrait;
++use MonsieurBiz\SyliusAdvancedShippingPlugin\Entity\ShippingTypeAwareInterface;
++use MonsieurBiz\SyliusAdvancedShippingPlugin\Entity\ShippingTypeAwareTrait;
+use Sylius\Component\Core\Model\ShippingMethod as BaseShippingMethod;
+use Sylius\Component\Shipping\Model\ShippingMethodTranslationInterface;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="sylius_shipping_method")
+ */
+#[ORM\Entity]
+#[ORM\Table(name: 'sylius_shipping_method')]
+-class ShippingMethod extends BaseShippingMethod
++class ShippingMethod extends BaseShippingMethod implements AddressProviderAwareInterface, ShippingTypeAwareInterface
+{
++    use AddressProviderAwareTrait;
++    use ShippingTypeAwareTrait;
++
+    protected function createTranslation(): ShippingMethodTranslationInterface
+    {
+        return new ShippingMethodTranslation();
+    }
+}
+```
+
+## Update your database schema
+
+Update your database schema with the plugin migrations:
+
+```bash
+bin/console doctrine:migrations:migrate
+```
+
+Generate the migration and update your database schema with the new entities fields:
+
+```bash
+bin/console doctrine:migrations:diff
+bin/console doctrine:migrations:migrate
+```
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏
Why did you propose to migrate or diff + migrate ?

Reply:
because the plugin has migrations